README.md

```diff
@@ -56,34 +56,33 @@ If your control panel is missing, feel free to [create an issue](https://github.
 - **Tested with control box**: CB38M2B(IB)-1
 - **Source**: Printed on the PCB of the control box.
   
-| RJ45 pin | Name       | Cable Color |
-| -------- | ---------- | ----------- |
-| 8        | RESET SWIM | Brown       |
-| 7        | PIN 20     | White       |
-| 6        | RX         | Purple      |
-| 5        | TX         | Red         |
-| 4        | GND1       | Green       |
-| 3        | +5V (VDD)  | Black       |
-| 2        | 29V+       | Blue        |
-| 1        | 29V-       | Yellow      |
+| RJ45 pin | Name       | Original Cable Color | Ethernet cable color (T568B) |
+| -------- | ---------- | -------------------- | ---------------------------- |
+| 8        | RESET SWIM | Brown                | White-Orange                 |
+| 7        | PIN 20     | White                | Orange                       |
+| 6        | RX         | Purple               | White-Green                  |
+| 5        | TX         | Red                  | Blue                         |
+| 4        | GND1       | Green                | White-Blue                   |
+| 3        | +5V (VDD)  | Black                | Green                        |
+| 2        | 29V+       | Blue                 | White-Brown                  |
+| 1        | 29V-       | Yellow               | Brown                        |
 
 #### [HS01B-1](https://www.loctekmotion.com/shop/control-panel/hs05a-1/)
 
 - **Desk model**: Flexispot E5B
 - **Tested with control box**: unknown
```

Comment:
@staxDB do you perhaps know which control box you have?

Reply:
I have the model CB38M2A-1